cloud_governance/policy/empty_buckets.py

```diff
@@ -6,6 +9,8 @@
 
 class EmptyBuckets(NonClusterZombiePolicy):
 
+    BUCKET_AGE = 30
```

Comment:
Do we want to wait 30 days till delete empty bucket?

Reply:
No, But we want to give some time to the users

Reply:
30 days, is it not too much?

Reply:
Yes, 
How about giving 7 days?
Any activities can be recorded in one week. So we can delete it if empty.

Reply:
As you wish, I think we can delete it immediatly. Because it related to cluster and also empty. Right?

Reply:
But, it's good to give some time.